src/components/sdk/MeetingControls/docs/ContentShareControl.stories.mdx

```diff
@@ -4,13 +4,15 @@ import ContentShareControl from '../ContentShareControl';
 <Meta title="SDK Components/MeetingControls/ContentShareControl" />
 
 # ContentShareControl
-Renders a `ControlBarButton` with pop over menu options to pause/play content share once enabled.
 
-On clicking this button, the content share toggles between on/off state.
+The `ContentShareControl` component renders a `ControlBarButton` with pop over menu options to pause/play content share once enabled.
```

Comment:
The \`ContentShareControl\` component renders a \`ControlBarButton` along with pop over menu options only when the current user is sharing content.